apps/challenges/serializers.py

```diff
@@ -15,12 +15,40 @@
     PWCChallengeLeaderboard,
     StarChallenge,
     UserInvitation,
+    ChallengeTags,
 )
 
 
+class ChallengeTagsSerializer(serializers.ModelSerializer):
+    """
+    Serialize the ChallengeTags Model.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(ChallengeTagsSerializer, self).__init__(*args, **kwargs)
+        context = kwargs.get("context")
+        if context:
+            tag_name = context.get("tag_name")
+            if tag_name:
+                kwargs["data"]["tag_name"] = tag_name
+            challenge = context.get("challenge")
+            if challenge:
+                kwargs["data"]["challenge"] = challenge.pk
+
+    class Meta:
+        model = ChallengeTags
+        fields = "__all__"
+
+
 class ChallengeSerializer(serializers.ModelSerializer):
 
     is_active = serializers.ReadOnlyField()
+    list_tags = ChallengeTagsSerializer(many=True)
```

Comment:
Why are we doing this?

Reply:
We get the Id or index of list_tags as it is a manytomany in challenge. So we serialize it to get the value or name of tag